File: push/templates/app-java/src/main/java/_DefaultPushReceiver.java
package <%= appPackage %>.services.push;

import android.app.Activity;
import android.content.BroadcastReceiver;
import android.content.ComponentName;
import android.content.Context;
import android.content.Intent;

public class DefaultPushReceiver extends BroadcastReceiver {
    public DefaultPushReceiver() {
    }

    @Override
    public void onReceive(Context context, Intent intent) {
        Intent push = new Intent(context, PushIntentService.class);
        push.putExtras(getResultExtras(true));
        context.startService(push);
        setResultCode(Activity.RESULT_OK);
    }
}

File: push/templates/app-kotlin/src/main/java/_DefaultPushReceiver.kt
package <%= appPackage %>.services.push;

import android.app.Activity;
import android.content.BroadcastReceiver;
import android.content.ComponentName;
import android.content.Context;
import android.content.Intent;

class DefaultPushReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        val push = Intent(context, PushIntentService::class.java)
        push.putExtras(getResultExtras(true))
        context.startService(push)
        resultCode = Activity.RESULT_OK
    }
}